routes/cube_routes.js

```diff
@@ -1389,6 +1344,120 @@ async function bulkuploadCSV(req, res, cards, cube) {
   }
 }
 
+async function bulkuploadCSV(req, res, cards, cube) {
+  let changelog = '';
+  let { newCards, newMaybe, missing } = await bulkuploadCardsCSV(cards);
+  newCards.forEach((card) => (changelog += addCardHtml(carddb.cardFromId(card.cardID))));
+  cube.cards.push(...newCards);
+  cube.maybe.push(...newMaybe);
+  await generateBlogpost(req, res, cube, changelog, newCards.concat(newMaybe), missing);
+}
+
+function compareCubes(req, res, cubeA, cubeB) {
+  if (!cubeA) {
+    req.flash('danger', 'Base cube not found');
+    res.status(404).render('misc/404', {});
+  } else if (!cubeB) {
+    req.flash('danger', 'Comparison cube was not found');
+    res.redirect('/cube/list/' + cubeA._id);
+  } else {
+    return new Promise(function(resolve, reject) {
```

Comment:
Please use async/await - no callbacks, and no promises

Reply:
I hadn't realized there was a way to get prices without doing it this way. Updating it now.